src/common/dependencies.ts

```diff
@@ -1,13 +1,8 @@
-import semver from 'semver';
 import { Version } from './common-types';
-import { isArmMac, isMac, isWindows } from './env';
-
-const KB = 1024 ** 1;
-const MB = 1024 ** 2;
-const GB = 1024 ** 3;
 
 export interface PyPiPackage {
-    packageName: string;
+    displayName?: string;
```

Comment:
This is no longer optional.
```suggestion
    displayName: string;
```